esphome/components/i2c/__init__.py

```diff
@@ -2,13 +2,15 @@
 import esphome.config_validation as cv
 from esphome import pins
 from esphome.const import CONF_FREQUENCY, CONF_ID, CONF_SCAN, CONF_SCL, CONF_SDA, CONF_ADDRESS, \
-    CONF_I2C_ID
+    CONF_I2C_ID, CONF_CHANNEL, CONF_MULTIPLEXER
 from esphome.core import coroutine, coroutine_with_priority
 
 CODEOWNERS = ['@esphome/core']
 i2c_ns = cg.esphome_ns.namespace('i2c')
 I2CComponent = i2c_ns.class_('I2CComponent', cg.Component)
 I2CDevice = i2c_ns.class_('I2CDevice')
+I2CMultiplexer = i2c_ns.class_('I2CMultiplexer')
```

Comment:
I don't understand the amazing codegen enough to argue as to why the parent classes are always listed here as per the c++ code...

```suggestion
I2CMultiplexer = i2c_ns.class_('I2CMultiplexer', I2CDevice)
```